back/app/controllers/web_api/v1/groups_projects_controller.rb

```diff
@@ -7,14 +7,14 @@ def index
       .includes(:group)
 
     @groups_projects = case params[:sort]
-      when 'new'
-        @groups_projects.order_new
-      when '-new'
-        @groups_projects.order_new(:asc)
-      when nil
-        @groups_projects
-      else
-        raise 'Unsupported sort method'
+    when 'new'
```

Comment:
I like the indentation in this case, but I can definitely live with this change.